cmd/influxd/run/server_test.go

```diff
@@ -891,7 +891,7 @@ func TestServer_Query_Tags(t *testing.T) {
 		&Query{
 			name:    "tag without field should return error",
 			command: `SELECT host FROM db0.rp0.cpu`,
-			exp:     `{"results":[{"error":"select statement must include at least one field"}]}`,
+			exp:     `{"results":[{}]}`,
```

Comment:
Obviously this test existed before this change, and I just modified the expected output. The case you do mention is tested elsewhere.